uwb/example/lib/screens/game_screen.dart

```diff
@@ -0,0 +1,382 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:uwb/uwb.dart';
+
+import '../game_widgets/ball.dart';
+import '../game_widgets/brick.dart';
+import '../game_widgets/brick_field.dart';
+import '../game_widgets/paddle.dart';
+import '../game_widgets/score.dart';
+import '../game_widgets/screen_overlay.dart';
+
+/// The [BreakoutGame] class.
+///
+/// Contains the code for the breakout game.
+class BreakoutGame extends StatefulWidget {
+  /// The [BreakoutGame] constructor.
+  const BreakoutGame({Key? key}) : super(key: key);
+
+  @override
+  BreakoutGameState createState() => BreakoutGameState();
+}
+
+/// Contains the direction enums of the ball.
```

Comment:
An enum doesn't contain anything, it is a type specification which describes the different values a variable can have. Therefore the current description is confusing, a better description would be:

```suggestion
/// The possible directions the ball can move in.
```